README.md

```diff
@@ -398,6 +398,30 @@ setTimeout(function() {
   proxy.removeResolver(topPriority);
 }, 600000);
 ```
+### Modify Request body
+
+bodyChange options parameter let you modify the incoming req.body. An example is below:
+
+```javascript
+ // for every URL path that starts with /api/, send request to upstream API service
+ //Warning not handling promise. 
+ var bodyChange = function(proxyReq, bodyOrigin) {
+   bodyOrigin.modify = true;
+   //if you change content-type => proxyReq.setHeader('content-type', 'application/xml');
+   delete bodyOrigin.xxx;
+ };
+
+
+ var proxy = new require('redbird')({
+    port: 8080,
+    body: {
+      change: bodyChange,
+      //parser: bodyParserXMl import custom body-parser compatible with body-parser
+      parserName: 'json' // based on body-parser text||raw||json||urlencoded
```

Comment:
this is a option specific to ```body-parser```, I do not think it should be sent as an option, instead create the parser for the expected type of data and pass that as a "parser" in the "parser" option.